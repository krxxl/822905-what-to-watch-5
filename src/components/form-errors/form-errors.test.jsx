import React from "react";
import renderer from "react-test-renderer";
import FormErros from "./form-errors";

describe(`Render FormErrors`, () => {
  it(`Render FormErrors`, () => {
    const tree = renderer
      .create(
          <FormErros formErrors={`Error`}/>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
