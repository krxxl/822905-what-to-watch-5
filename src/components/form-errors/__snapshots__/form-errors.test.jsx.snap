// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render FormErrors Render FormErrors 1`] = `
<p>
  Error
</p>
`;
